.circleci/config.yml

```diff
@@ -1651,13 +1605,16 @@ workflows:
           docker_file: op-challenger/Dockerfile
           docker_name: op-challenger
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          load_base_image: "op_stack_go"
           docker_context: .
+          requires: ['op-stack-go-docker-build']
+          build_args: --build-arg OP_STACK_GO_BUILDER="$IMAGE_BASE_PREFIX/op_stack_go:<<pipeline.git.revision>>"
           platforms: "linux/amd64,linux/arm64"
+          publish: true
+          release: true
           context:
             - oplabs-gcr-release
-          requires:
-            - hold
-      - docker-build:
+      - docker-build:  # TODO: this was a "build" but not a release. Should this be a "release"?
```

Comment:
i think yes we should include this in both paths. not sure why this would be release specific, i think its mainly there to prevent dns related flakes.